pm_muairss_write/get_out_folder.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
You can't add comments to empty files so I'm putting it here: does the `initial` file above do anything?

Reply:
no, it was just to resolve some conflict occuring in creating a new branch(needed initial commit changes)